geomstats/_backend/pytorch/__init__.py

```diff
@@ -501,6 +535,14 @@ def mean(x, axis=None):
     return torch.mean(x, dim=axis)
 
 
+def where(condition, x, y):
+    if not torch.is_tensor(x):
+        x = torch.tensor(x)
+    if not torch.is_tensor(y):
+        y = torch.tensor(y)
+    return torch.where(condition, x, y)
```

Comment:
I ran into errors trying to do it like this the other day:
- if x and y are None
-

Reply:
Good points!

- I added the case where x and y are None
- This is because torch.where only work if x and y are of same `dtype`. Same with a lot of torch functions actually. The function now casts y with the`dtype` of x.

I have added your tests and they pass. Thanks!